controls/SV-261916.rb

```diff
@@ -52,10 +52,12 @@
     connection_error_regex = Regexp.new(connection_error)
 
     sql_result = sql.query(security_definer_sql, [database])
-
+    privs = input('priv_esc')
     describe.one do
       describe sql_result do
-        its('output') { should eq '' }
+        if its('output') != ''
+          its('output') { should include privs }
```

Comment:
Then in the logic, the need would be to check that a given user has been permitted privileged escalation.